src/Components/RosTable/RosTable.js

```diff
@@ -64,7 +64,7 @@ class RosTable extends React.Component {
                         id: index,
                         isOpen: false,
                         cells: [
-                            { title: actionLink(id, row.display_name, 'system-link', '#') },
+                            { title: actionLink(id, row.display_name, 'system-link', 'insights/ros/:systemId'), component={RosDetailsPage} },
```

Comment:
This should have the `systemId` in URL (is the ID system UUID? @kgaikwad)
```suggestion
                            { title: actionLink(id, row.display_name, 'system-link', `insights/ros/${id}`), component={RosDetailsPage} },
```

Reply:
@karelhala, 

Background -
According to previous DB schema, `id` column was the primary key of `PerformanceProfile` table.
But with new DB changes, it removed `id` column from this table and added `system_id` column which is foreign key of `systems` table where we stored information like `inventory_id`.
For Reference - https://github.com/RedHatInsights/ros-backend/commit/8322833f04fd7fbba36712141feefcbfdc582810#diff-3ca3958b01e8a917a7623495aa440543cfd9b2ce07daa94695aad9c20d9b0ff0L10



> is the ID system UUID? @kgaikwad

No, it wasn't UUID and currently `system_id` also not a UUID field.

Reply:
Due to new DB schema changes, the API was broken. 
PR opened on backend side which will fix the API.
Link - https://github.com/RedHatInsights/ros-backend/pull/36

Reply:
Opened new [PR-21](https://github.com/RedHatInsights/ros-frontend/pull/21) which replaces `id` key with `inventoryId` as per new response changes on backend.

Reply:
`insights_id` is not the right parameter either, it referes to insights reporter, you can have systems without such id if this system comes from different souce. The ID you are interested in is ID from this request https://cloud.redhat.com/docs/api/inventory#operations-hosts-api\.host\.get_host_list. This ID must come from inventory service and can't be asutogenerated on ROS backand, otherwise there won't be a proper link between inventory system and ROS data.

Reply:
Yeah, `inventoryId` is nothing but a `id` value that is coming from inventory service and not autogenerated on ROS backend.
column value from `hosts` table.

Reply:
updated it to `insights/ros/${inventoryId}`